tracim/tracim
# Fix(frontend)  chrome performance workaround

To enable correct performance when base64 images are inserted.

https://github.com/tracim/tracim/issues/4591
https://github.com/tracim/tracim/issues/4611

Old hack creates new bugs : https://github.com/tracim/tracim/issues/6224

Which is fixed in this PR but there is a bug where when editing a content the popup will open empty ( seems to happen after reloading the page (normal reloading not removing cache) and if the devtools are closed )

How to reproduce : (on Firefox only )
- Go on a page where edit thread content is available 
- Have the devtools closed
- Reload the page
- Click on edit from a message 

This bug didn't seem to appear when the devtools are open or it's the first loading of tracim or when an alert is placed is the code before; it's possible that the bug cause is because the property of the coment are set too late.

@floreal15 todo: add more details about the issue. We will see what to do depending on whether we merge this PR or not.


## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**
